packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -885,6 +885,28 @@ module.exports.external = [
       }
     },
   },
+  {
+    name: "trimAndLower",
+    description: "Trims and converts string to lower case: trimAndLower string",
+    func: function (o) {
```

Comment:
nit: `o` usually refers to object AFAIK, especially with this being js, I'd push for more descriptive variable names

ie this would be `str` instead of `o`

Reply:
sounds good. thats just the way it was before in the helpers so I followed that, but will change to str